salt/modules/systemd.py

```diff
@@ -158,6 +160,123 @@ def systemctl_reload():
     return retcode == 0
 
 
+def _default_runlevel():
+    '''
+    Try to figure out the default runlevel.  It is kept in
+    /etc/init/rc-sysinit.conf, but can be overridden with entries
+    in /etc/inittab, or via the kernel command-line at boot
+    '''
+    # Try to get the "main" default.  If this fails, throw up our
+    # hands and just guess "2", because things are horribly broken
+    try:
+        with salt.utils.fopen('/etc/init/rc-sysinit.conf') as fp_:
+            for line in fp_:
+                if line.startswith('env DEFAULT_RUNLEVEL'):
+                    runlevel = line.split('=')[-1].strip()
+    except Exception:
+        return '2'
```

Comment:
except you want to pollutate the logs, i dont think we should add a log here as it is a sane default anyway.
If we do, upstart module needs also to be patched.